smsclient_core/smsclient_view.xml

```diff
@@ -0,0 +1,194 @@
+                            <field name="sms_account_visible" invisible="1"/>
+                            <field name="login_provider" attrs="{'invisible': [('login_provider_visible', '=', False),]}"/>
+                            <field name="login_provider_visible" invisible="1"/>
+                            <field name="password_provider" attrs="{'invisible': [('password_provider_visible', '=', False),]}"/>
+                            <field name="password_provider_visible" invisible="1"/>
+                            <field name="from_provider" attrs="{'invisible': [('from_provider_visible', '=', False),]}"/>
+                            <field name="from_provider_visible" invisible="1"/>
+                            <field name="state"/>
+                            </group>
+                        </page>
+                        <page string="Permission">
+                            <separator string="Access Permission"
+                                colspan="4" />
+                            <field name="users_id" select="1"
+                                colspan="4" nolabel="1" />
+                        </page>
+                        <page string="Additional option">
+   		                	<group>
+	           		          	<group>
+	   		          	        	<field name="validity" attrs="{'invisible': [('validity_visible', '=', False),]}"/>
+                                    <field name="validity_visible" invisible="1"/>
+	   		          	        	<field name="classes" attrs="{'invisible': [('classes_visible', '=', False),]}"/>
+                                    <field name="classes_visible" invisible="1"/>
+	   		          	        	<field name="deferred" attrs="{'invisible': [('deferred_visible', '=', False),]}"/>
+                                    <field name="deferred_visible" invisible="1"/>
+	   		          	        	<field name="nostop" attrs="{'invisible': [('nostop_visible', '=', False),]}"/>
+                                    <field name="nostop_visible" invisible="1"/>
+	           		          	</group>
+	   		          	        <group>
+		                   			<field name="priority" attrs="{'invisible': [('priority_visible', '=', False),]}"/>
+                                    <field name="priority_visible" invisible="1"/>
+		                   			<field name="coding" attrs="{'invisible': [('coding_visible', '=', False),]}"/>
+                                    <field name="coding_visible" invisible="1"/>
+		                   			<field name="tag" attrs="{'invisible': [('tag_visible', '=', False),]}"/>
+                                    <field name="tag_visible" invisible="1"/>
+		                   			<field name="char_limit" attrs="{'invisible': [('char_limit_visible', '=', False),]}"/>
+                                    <field name="char_limit_visible" invisible="1"/>
+								</group>
+							</group>
+						</page>
```

Comment:
Please fix the indentation of this file there are a bunch of tab we only want spaces